channeldb/migrations_test.go

```diff
@@ -0,0 +1,191 @@
+package channeldb
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	"testing"
+
+	"github.com/coreos/bbolt"
+)
+
+// TestPaymentStatusesMigration checks that already completed payments will have
```

Comment:
Nice! I forget at times that we have the infra to test the pre and post migration state.

Reply:
@vapopov's idea :)